status/types/message.nim

```diff
@@ -207,10 +207,11 @@ proc toMessage*(jsonMsg: JsonNode): Message =
     # the "Message" is contained as an address in the wallet of logged in user. If yes, means that
     # currently logged in user has initiated a transaction (he is a sender), otherwise currently 
     # logged in user is a recipient.
-    message.isCurrentUser = currentUserWalletContainsAddress(message.commandParameters.fromAddress)
+    if message.commandParameters.fromAddress != "":
```

Comment:
What happens if `fromAddress` is empty? by default `isCurrentUser` is false?

Reply:
hey @borismelnik  in that case it will be as per ln 148 -
      isCurrentUser: publicChatKey == jsonMsg{"from"}.getStr,

Reply:
@borismelnik can you please review when you got the time so that I can merge it.